### PR TITLE
New test cases

### PR DESCRIPTION
**Pull request by velibor7**
I just need to make sure that the convention is okay so I can finish other tests. Bank IP in the file is 10.2.3.4:80 so I kept it that way, but I can change it :)

### CODE REVIEW

**Comment by shahraizali** [comment]
yeah, this looks fine 👍 . keep the IP what it already is no need to change. I'll do a final review before merging you can finish up others as well. maybe someone else can have a look as well but don't wait on the response fix others already.

File: tests/banks.py
Comment (shahraizali) [inline tests/banks.py:25]: If others don't have next and prev let's remove from here as well.

Comment (velibor7) [reply tests/banks.py]: Done with 59bc9f3

**Comment by velibor7** [comment]
With 00e988e I did the same thing @VincentHch did with #47, I just made it consistent with the rest of my code :)

**Review by CirXe0N — CHANGES_REQUESTED** [review]
Make sure once the PR https://github.com/thenewboston-developers/thenewboston-python-client/pull/44 is merged to also the change the typings of the related methods. 

The PR https://github.com/thenewboston-developers/thenewboston-python-client/pull/44 will be merged before this PR.

**Comment by VincentHch** [comment]
> Make sure once the PR #44 is merged to also the change the typings of the return values of the related methods.
> 
> The PR #44 will be merged before this PR.

@velibor7 basically just change every `list` as return type of method by `dict`

**Comment by shahraizali** [comment]
Okay, the other PR has been merged. It had added annotations but since previously the tests were returning arrays so the merged PR has annotation mentioned as `list` we would want to change that to `dict` like @VincentHch mentioned. You can also verify on your end as well that they are actually 'dict'.

File: tnb/banks.py
Comment (CirXe0N) [inline tnb/banks.py:250]: Please also add typings to the parameters.

```suggestion
    def post_block(self, account_number: str, balance_key: str, transactions: list, signature: str) -> dict:
```

**Review by CirXe0N — CHANGES_REQUESTED** [review]
Only one small change needed. The rest looks good.

**Comment by CirXe0N** [comment]
@shahraizali @sbalasa Why do we have a `dist` folder with a `thenewboston-python-client-0.0.2.tar.gz` file in this project?

**Comment by velibor7** [comment]
I'm nooot sure. Should I add it in gitignore?

**Comment by shahraizali** [comment]
> Why do we have a `dist` folder

That is the distribution package. added deliberately by bucky multiple times. we will be updating it as well once we do another release.

https://github.com/thenewboston-developers/thenewboston-python-client/commit/eed0b3c2417975e75537dffa42a1afc351c8ebf8

https://github.com/thenewboston-developers/thenewboston-python-client/commit/dd791575b0c9646cbee20df53d24190869d4c212

**Comment by shahraizali** [comment]
This is where we have it connected to. 
https://pypi.org/project/thenewboston-python-client/#history

**Comment by CirXe0N** [comment]
@shahraizali Ok, if it is for pypi. We don't need the `dist` folder. We can just update the `setup.py` to download directly from Github.
(https://github.com/thenewboston-developers/thenewboston-python-client/archive/master.zip)

Later on we can create releases within Github. (https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository)

**Review by CirXe0N — CHANGES_REQUESTED** [review]
Another small change :) Please remove the `.tar.gz` from the PR

**Comment by velibor7** [comment]
Hmmm, I did that, but the code wil l generate a new one when someone else runs it. If we don't need dist at all I think its better to put it in ignore? Also now that I deleted the tar gz does it effect our release on pypi?

**Comment by CirXe0N** [comment]
@velibor7 I meant that you just remove it from this PR. In others words, not make any changes to the file. I did not mean for you to delete the file from the project. We will look into that in a new issue.

**Comment by velibor7** [comment]
@CirXe0N So just to be clear so I don't mess something up. I should add another commit with tar.gz, and then delete it here on this pr?

**Comment by CirXe0N** [comment]
@velibor7 Add the *original* tar.gz back into the dist folder. You can do this by doing the following:
1. Download the file from here (https://github.com/thenewboston-developers/thenewboston-python-client/raw/master/dist/thenewboston-python-client-0.0.2.tar.gz)
2. Place the file in the dist folder.
3. Make a new commit with this file staged.

**Comment by shahraizali** [comment]
It should only generate dist if we intend to generate it. like running `sdist` command not sure why is it generating it for you automatically.

**Review by CirXe0N — APPROVED** [review]

**Comment by velibor7** [comment]
It does not generate automatically, sorry for misunderstanding :)

**Pull request merged**
